Streams are not properly closed

bsulich2 opened the pull request:
Refactoring classes to handle autoclosable inside try with resources block.

Fixes https://github.com/bndtools/bnd/issues/1929

Signed-off-by: Bartlomiej Sulich <bartlomiej.sulich@intel.com>

bjhargrave commented on biz.aQute.bndlib/src/aQute/bnd/osgi/Builder.java:
```diff
@@ -113,8 +113,7 @@ public Jar build() throws Exception {
-					InputStream in = new FileInputStream(mff);
+				try (InputStream in = new FileInputStream(mff)){
 					manifest = new Manifest(in);
 					in.close();
```
There is already a close here. If you want to use try-with-resource, then remove the explicit close call.

bjhargrave commented:
You should also fix the commit comment to say 'Streams' and not 'Stream' :-)

bsulich2 commented:
- fixed comment. 
- added try with resources to [Workspace.java](https://github.com/bsulich2/bnd/blob/1929_refactor_streams_close/biz.aQute.bndlib/src/aQute/bnd/build/Workspace.java#L688)
- removed close() from try with resources block in [Builder.java](https://github.com/bsulich2/bnd/blob/1929_refactor_streams_close/biz.aQute.bndlib/src/aQute/bnd/osgi/Builder.java#L117)
